frontend/src/lib/lemon-ui/LemonTextArea/LemonTextArea.tsx

```diff
@@ -87,14 +99,19 @@ export const LemonTextArea = React.forwardRef<HTMLTextAreaElement, LemonTextArea
                 }}
                 {...textProps}
             />
-            {textProps.maxLength !== undefined ? (
-                <div
-                    className={cn(
-                        'flex flex-row justify-end text-sm',
-                        textLength >= textProps.maxLength && 'text-error'
-                    )}
-                >
-                    {textLength} / {textProps.maxLength}
+            {footer || textProps.maxLength ? (
```

Comment:
this is a little messy but lets me reduce the amount of whitespace below the textbox

when i'm adding emoji picker i want to revisit all of this and add a mini toolbar to the text area